[DOCS-1407] Improve the accessibility of link text (#1438)
wandb/docs: content/en/guides/core/registry/model_registry/_index.md

   * [Protected models and access control]({{< relref "./access_controls.md" >}}).
   * [How to connect Registry to CI/CD processes]({{< relref "/guides/core/automations/" >}}).
   * Set up [Slack notifications]({{< relref "./notifications.md" >}}) when a new model version is linked to a registered model.

> * Review [What is an ML Model Registry?](https://wandb.ai/wandb_fc/model-registry-reports/reports/What-is-an-ML-Model-Registry---Vmlldzo1MTE5MjYx) report on how the Model Registry fits into your ML workflow and the benefits of using one for model management. 

Author: ```suggestion
* Review [What is an ML Model Registry?](https://wandb.ai/wandb_fc/model-registry-reports/reports/What-is-an-ML-Model-Registry---Vmlldzo1MTE5MjYx) to learn how to integrate Model Registry into your ML workflow.
```
